eng/scripts/get_package_properties.py

```diff
@@ -11,6 +11,8 @@
     parser.add_argument('-s', '--search_path', required=True, help='The scope of the search')
     args = parser.parse_args()
 
-    for p in glob.glob(args.search_path, recursive=True):
```

Comment:
What about this doesn't work for python 2.7?

Reply:
Also why do we care about 2.7 support in this script? Is there a case we are running in that context?

Reply:
`glob.glob( ,recursive=True)` does not work in python 2.7. This causes it to fail when it is run on some test pipelines that are using python 2.7